roles/proxy/defaults/main.yml

```diff
@@ -26,7 +26,6 @@ proxy_worker_rlmmiit_nofile: 30000
 proxy_worker_connections: 20000
 proxy_server_names_hash_bucket_size: 256
 
-proxy_athene_baseurl:
 proxy_primary_node_url:
```

Comment:
I believe we can also remove the `proxy_primary_node_url` since we don’t use it outside of the nginx deployment (?).
```suggestion

```

Reply:
Sounds correct to me 👍